### DISCUSSION TITLE
longhorn Recurrring Job backup is not working

### DISCUSSION THREAD

**Discussion by omgupta240909** [longhorn/longhorn#7337] [General]
time="2023-12-12T07:00:33Z" level=error msg="Failed to get recurring job bkp."
 error="Get \"https://clusterip:port/apis/longhorn.io/v1beta2/namespac es/longhorn-system/recurringjobs/bkp\": dial tcp clusterip:port i/o timeout"

**Comment by mantissahz** [comment]
Hi @omgupta240909,
Could you provide more details about your cluster including how many nodes, k8s version, os distro, network configuration, and so on?
It looks like a network issue from this message `dial tcp clusterip:port i/o timeout`.
That would be appreciated if you could provide the support bundle.

**Comment by omgupta240909** [comment]
Hi @mantissahz  The k8s cluster having 9 worker 3 master node, and for the backup we are having nfs4 of  size 2Tb , we do take daily  backup of volume using the recurring job , but for the few days, it was observing that the recurring job backup is not completing. 
k8s version 1.27 and container runtime is crio , os version is ubuntu 22.04  all that is on-perm.'
Name of the reccuring job is "bkp" 
and the log of the pod is 
time="2023-12-12T07:00:33Z" level=error msg="Failed to get recurring job bkp."
error="Get "https://pod subnet ip :443/apis/longhorn.io/v1beta2/namespac es/longhorn-system/recurringjobs/bkp": dial tcp po subnet ip:443 i/o timeout"

**Comment by mantissahz** [comment]
Any upgrade for k8s or Longhorn?
Can you check if the recurring job `bkp` exists by `kubectl -n longhorn-system get lhrj`?
Is `pod subnet ip` a domain name or an IP address?
And could you restart the `longhorn-manager` pods and check if this happened again?
It looks like that kube config is not correct.

**Comment by omgupta240909** [comment]
Hi @mantissah after restarting the pods of longhorn manger the backup job is working fine then created a new recurring job and applied to all the volumes.

Thanks.
